.drone.yml

```diff
@@ -8,7 +8,7 @@ steps:
   pull: never
   commands:
   - export RUST_BACKTRACE=1
-  - export RUSTFLAGS=-Ctarget-feature=+aes,+ssse3
+  - export RUSTFLAGS=-Ctarget-feature=+aes,+sse2,+sse4.1,+ssse3
```

Comment:
update the pipeline signature

Reply:
Updated the signature. Hopefully it'll work now.